.travis.yml

```diff
@@ -5,15 +5,14 @@ language: python
 
 dist: trusty
 
-python:
-  - "2.7"
-
 install:
   - pip install pylint tox scapy
   - pip install tornado==4.5.3
   - python setup.py install
 
 script:
+  - pyenv global 2.7.14 3.6.3
```

Comment:
This takes a lot of time so only do it if __python: 2.7__ does not work for your purposes.